66083.typ

```diff
@@ -515,13 +515,14 @@
 #set text(fallback: false, font: "DejaVu Sans")
 
 #let heading_size = 10pt
-#let margins = 5mm
+#let margins = 4mm     // Margins around the outer edges of the printed sheet.
+#let crease_gap = 2mm  // Size of the gap that will be cut for the crease.
```

Comment:
I'm curious if these spacing values should be moved out into the common file? Or does it make sense to keep them per-checklist?

Reply:
I'm not really sure myself, I flip-flopped a bit on it.

That said, the conceptual design is that the bottom part of each airplane's `.typ` file contains the page-level formatting of that checklist. The common file only concerns itself with the formatting of individual content boxes, and every single size in that entire file is relative to the text size. So keeping these here is consistent with the overall design.

*shrug*